data/mods/innawood/npcs/items_generic.json

```diff
@@ -218,5 +218,23 @@
       [ "yeast", 5 ],
       [ "zucchini", 3 ]
     ]
+  },
+  {
+    "id": "survivor_bashing",
+    "//": "Override for hardcoded reference in npc::starting_weapon",
+    "type": "item_group",
+    "items": [ [ "cudgel", 1 ], [ "stick_long", 2 ], [ "rock", 5 ] ]
+  },
+  {
+    "id": "survivor_stabbing",
+    "//": "Override for hardcoded reference in npc::starting_weapon",
+    "type": "item_group",
+    "items": [ [ "pointy_stick", 5 ] ]
```

Comment:
```suggestion
    "items": [ [ "pointy_stick", 5 ], [ "pointy_stick_long", 2 ] ]
```

Reply:
These all look good, thanks. Honestly I was having trouble coming up with weapons that were appropriate but not super well-crafted and fit with the primitivist style. I probably should have spent more time looking at the crafting menu.

These weapons can be doled out to any class that doesn't have a defined `weapon_override` or otherwise spawn with a weapon. It chooses a group based on the NPC's highest skill on spawn, and then hands it a weapon. In DDA the groups are a pretty wide assortment of improvised weaponry, it's hard for me to make a direct comparison given the generally lower 'power level' in innawoods. I'll link the JSON in case you had any trouble finding it:

https://github.com/CleverRaven/Cataclysm-DDA/blob/3fc7b38ea8a67116e08f77739c4611f496b39a3e/data/json/monsterdrops/zombie_survivor.json#L105

Reply:
Oh, you may also be interested in seeing #65592. I remember reading some old comments from you that you were having trouble getting rid of the lighters? I can't blame you, random hardcoded insertion in starting inventory that only appears in very specific circumstances.